src/components/controls/gamepad-controls.js

```diff
@@ -0,0 +1,110 @@
+var registerControls = require('../../core/controls').registerControls;
```

Comment:
The con is true, but like you said the gamepad wouldn't necessarily be used for first-person movement/rotation. Even in third-person games, like a platform, the gamepad behaves differently (left stick controls both rotation + velocity). So it might be another tough choice to leave it non-prescriptive. But I think the gamepad could be used for lots and lots of other things.

Well, then we only ship HMD + mousedrag, which are about the minimum + non-prescriptive we can get I think.